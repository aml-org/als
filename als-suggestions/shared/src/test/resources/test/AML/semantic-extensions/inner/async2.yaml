asyncapi: "2.0.0"
info:
  title: semantic test
  version: 1
x-maintainer:
    *