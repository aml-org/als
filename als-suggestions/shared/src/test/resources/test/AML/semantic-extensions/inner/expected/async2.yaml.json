[
  {
    "label": "email",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "email",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11email",
    "filterText": "email:\n  ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 5,
          "character": 4
        },
        "end": {
          "line": 5,
          "character": 4
        }
      },
      "newText": "email:\n  "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "enumTest",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "enumTest",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11enumTest",
    "filterText": "enumTest: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 5,
          "character": 4
        },
        "end": {
          "line": 5,
          "character": 4
        }
      },
      "newText": "enumTest: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "name",
    "kind": [
      
    ],
    "detail": "docs",
    "documentation": "name",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11name",
    "filterText": "name: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 5,
          "character": 4
        },
        "end": {
          "line": 5,
          "character": 4
        }
      },
      "newText": "name: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]